test/convert.test.ts

```diff
@@ -38,7 +50,11 @@ describe("Convert", () => {
     })
 
     it("does not encode Windows drive specifiers", () => {
-      expect(Convert.pathToUri("d:\\ee\\ff.txt")).toBe("file:///d:/ee/ff.txt")
+      // This test only succeeds on windows. (Because of the difference in the processing method of drive characters)
+      // However, it is enough to test the windows drive character only on windows.
+      if (process.platform === "win32") {
+        expect(Convert.pathToUri("d:\\ee\\ff.txt")).toBe("file:///d:/ee/ff.txt")
+      }
```

Comment:
```suggestion
      // This test only succeeds on windows. (Because of the difference in the processing method of drive characters)
      // However, it is enough to test the windows drive character only on windows.
      if (process.platform === "win32") {
        pending("Only test on windows")
      }
      expect(Convert.pathToUri("d:\\ee\\ff.txt")).toBe("file:///d:/ee/ff.txt")
```

Reply:
Thank you for your review.
Fixed to use pending ().

However, the test is failing on unrelated changes.